subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/AbstractBuildExperimentRunner.java

```diff
@@ -75,7 +75,10 @@ private Profiler createProfiler(String profilerName) {
         optionParser.accepts("profiler");
         ProfilerFactory.configureParser(optionParser);
         ProfilerFactory profilerFactory = ProfilerFactory.of(Collections.singletonList(profilerName));
-        return profilerFactory.createFromOptions(optionParser.parse());
+        String[] options = profilerName.startsWith("jprofiler")
```

Comment:
Why use `startsWith()` instead of `equals()`?

Reply:
We could have a similar way to specify heap or cpu profiling as we do with `async-profiler`/`async-profiler-heap`. Though you are right, for now I can use `equals`.

Reply:
I am using `equals` now.